include/alpaka/mem/view/ViewPlainPtr.hpp

```diff
@@ -82,14 +82,24 @@ namespace alpaka
                 {}
 
                 //-----------------------------------------------------------------------------
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC
                 ViewPlainPtr(ViewPlainPtr const &) = delete;
                 //-----------------------------------------------------------------------------
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC
                 ViewPlainPtr(ViewPlainPtr &&) = default;
                 //-----------------------------------------------------------------------------
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC
                 auto operator=(ViewPlainPtr const &) -> ViewPlainPtr & = delete;
                 //-----------------------------------------------------------------------------
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC
                 auto operator=(ViewPlainPtr &&) -> ViewPlainPtr & = default;
                 //-----------------------------------------------------------------------------
+                ALPAKA_NO_HOST_ACC_WARNING
+                ALPAKA_FN_HOST_ACC
```

Comment:
Who is using the `ViewPlainPtr` in device code and where? Until now my impression was that the views/bufs are a host only thing (for now) and we pass raw pointers+size into the kernels. You could change this while resolving the merge conflicts here.

Reply:
ok, I do not know, if this is needed on device at all, but I would follow your thoughts and change the annotation to host only. Tested it already and everything passed.